config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_iamauditconfigs.iam.cnrm.cloud.google.com.yaml

```diff
@@ -81,12 +82,39 @@ spec:
               resourceRef:
                 description: Immutable. Required. The GCP resource to set the IAMAuditConfig
                   on (e.g. project).
+                oneOf:
```

Comment:
@maqiuyujoyce I think this is technically backwards incompatible but I figured this was supposed to be in to begin with.

happy to take it out if you think we shouldn't

Reply:
IAM is in general fairly critical so let's try to keep it backwards compatible for now.